### PR TITLE
[Term Entry] JavaScript Dates .getUTCMonth() #2426 

### LINKED ISSUE #2426: [Term Entry] JavaScript Dates .getUTCMonth()
### Reason/inspiration (optional)

We would like a new term entry for JavaScript [Dates](https://www.codecademy.com/resources/docs/javascript/dates) about the `.getUTCMonth()` method. The entry should go in a new file in `docs/content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md`

The entry should
    - Have an introductory paragraph about what the method is and does.
    - An example showing the method in use.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Iampetrus**
### Description

- Added .getUTCMonth() method for Javascript
- Closes #2426
- 
### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+Title:
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:2]: Please review another entry or template and update the metadata to match. There are several edits that are required:

- all of the content must be within single quotes
- the title and description should not begin on a separate line
- catalogcontent should include two items

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+‘.getUTCMonth()'
+Description:
+‘Method of date instances that returns the month date according to universal time, as a zero-based value (where zero indicates the first month of the year).'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:5]:
```suggestion
'Returns the month of the passed date according to universal time, as a zero-based value.' 
```

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  - ‘Web Development’
+  - ‘Computer Science’
+Tags:
+  - ‘Date’
+  - 'Methods’
+CatalogContent:
+ - 'introduction-to-javascript'
+---
+
+get methods retrieve the date components based on the user’s local timezone settings. For increased control over the dates and times, you can use the **`.getUTCMonth()`** method, which are exactly the same as the `getMonth()` method, except they calculate the time based on the UTC (Coordinated Universal Time) standard.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:16]:
```suggestion
Get methods are used to retrieve date components based on the user’s local timezone settings. The **`.getUTCMonth()`** method returns the month of the date passed, similar to `.getMonth()`. However, `.getUTCMonth()` calculates the time based on the UTC (Coordinated Universal Time) standard.
```

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Syntax
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:18]: add a line break after the header

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+```
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:19]:
```suggestion
```pseudo
```

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+const today = new Date();
+const month = today.getUTCMonth();
+```
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:22]: add a line below the syntax

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+Example above assigns the month portion of the current `Date` to the variable `month`.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:23]: update this line to explain the function, the return value, etc in more detail.

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+## Example
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:25]: add an empty line after the header

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+```js
+const date1 = new Date('December 31, 1975, 23:15:30 GMT+11:00');
+const date2 = new Date('December 31, 1975, 23:15:30 GMT-11:00');
+
+console.log(date1.getUTCMonth());
+// expected output December: 11
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:31]: delete the comments and use a shell block instead

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Iampetrus Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@Iampetrus This all set for another review. I just made a couple of small edits mostly to correct spacing.

**Review by SSwiniarski — APPROVED** [review]
@Iampetrus I did a few punctuation edits on the metadata, otherwise it looks good to me.

**Pull request merged**